components/feature/media/src/main/java/mozilla/components/feature/media/service/MediaSessionServiceDelegate.kt

```diff
@@ -144,32 +148,18 @@ internal class MediaSessionServiceDelegate(
                 .build())
     }
 
-    private suspend fun updateNotification(sessionState: SessionState?) {
-        controller = sessionState?.mediaSessionState?.controller
-        val notificationId = SharedIdsHelper.getIdForTag(
-            context,
-            AbstractMediaSessionService.NOTIFICATION_TAG
-        )
-
-        val notification = notification.create(sessionState, mediaSession)
-
-        // Android wants us to always, always, ALWAYS call startForeground() if
-        // startForegroundService() was invoked. Even if we already did that for this service or
-        // if we are stopping foreground or stopping the whole service. No matter what. Always
-        // call startForeground().
+    private suspend fun startForegroundNotification() {
+        val notification = notification.create(null, mediaSession)
         service.startForeground(notificationId, notification)
```

Comment:
Is your goal here to build a "dummy" notification that we can display immediately and then update the notification later with the actual content? That sounds good, but maybe we should not go through all that code here, to avoid having `suspend` and `runBlocking()` in this sequence, which can cause other problems in the future.

Reply:
Understood.  I'll update so we create the notification with the same dummy content without going through the function that create real notification.